Mathlib/Tactic/FunProp/Theorems.lean

```diff
@@ -210,8 +210,7 @@ def getTheoremsForFunction (funName : Name) (funPropName : Name) :
 
 --------------------------------------------------------------------------------
 
-/-- General theorem about function property
-  used for transition and morphism theorems -/
+/-- General theorem about function property used for transition and morphism theorems -/
```

Comment:
pre-existing: the grammar doesn't check out, should be either "about a function property" or "function properties"
CC @lecopivo

Reply:
Fixed!